sklearn/covariance/tests/test_robust_covariance.py

```diff
@@ -39,6 +41,10 @@ def test_mcd():
     # 1D data set
     launch_mcd_on_dataset(500, 1, 100, 0.001, 0.001, 350)
 
+def test_fast_mcd():
```

Comment:
The test of this function does not reflect the actual check performed by this function: `test_fast_mcd_on_invalid_input` would be better for instance.